design/features/payment-links/version-1.md

```diff
@@ -0,0 +1,248 @@
+
+For this version we wanted to make it clear that this new task was ‘optional’, while allowing room for expansion later. 
+
+For this reason, we introduced a new heading within the first “Create your form” section. 
+
+We also left the total number of ‘completed tasks’ the same, totalling ‘9’. We believe this will help remove confusion or ambiguity about whether a payment link needs to be added to a form in order to make it live.    
+
+Section 1 in the task list says “1. Create your form” with these four tasks: 
+
+- “Edit the name of your form”
+- “Add and edit your questions”
+- “Add a declaration for people to agree to”
+- “Add information about what happens next”
+
+Below this summary list of tasks in section 1 is a new heading-level 3, titled “Optional tasks”. This heading is followed by a summary list with a single item saying “Add a link to a payment page on GOV.UK Pay”. This row has a grey tag with the word “Optional” in it. We decided that this task could only ever be in one of two states: ‘Optional’ or ‘Completed’.  
+
+### Add a link to a payment page on GOV.UK Pay
+
+![“Add a link to a payment page on GOV.UK Pay” titled page showing guidance to help form creators set up a payment link. Screenshot](./screenshots-v1/002-add-a-link-to-a-payment-page-on-govuk-pay.png)
+
+Beneath the heading “Add a link to a payment page on GOV.UK Pay” is some initial explanatory text. 
+
+> You can use GOV.UK Pay (opens in a new tab) to set up a payment page (called a ‘payment link’).  
+> Once it’s set up, you can copy and paste the payment link URL into the box below. This will add a payment page to the end of your form.
+
+The next section, titled “Setting up GOV.UK Pay”, outlines what’s needed before you can get started with creating a payment link. 
+
+> You’ll need to set up a GOV.UK Pay account if you do not already have one.
+> You’ll also need to talk to your organisation’s finance team to:
+> - make sure you’re allowed to take payments in this way
+> - set up a payment service provider (PSP)
+
+This is followed by an inset text component informing form creators about potential timing implications if they have not already started the GOV.UK Pay set up journey:
+
+> “It could be up to several months before you’re ready to take payments. This depends on the PSP arrangements for your organisation.”  
+
+The next section, “Creating your payment link in GOV.UK Pay”, outlines key steps to make sure that the payment link works correctly with GOV.UK Forms:  
+
+> Once you have a GOV.UK Pay account you can “add a new service” and start creating your payment link.
+
+There is a heading-level 3, “Set up your payment link to use a form’s unique reference number”, which provides more detail about the reference numbers that GOV.UK Forms uses. This includes how to make sure these are correctly processed with the payment link provided by form creators:  
+
+> GOV.UK Forms adds a unique 8-character reference to each form submission.
```

Comment:
@christophercameron-ixd is there a way to add line breaks between these rows - to reflect how the content is presented on the screen - while still keeping this showing as inset text for presentation purposes here? 

I tried adding a line break but that ended up presenting it as lots of separate pieces of content (with a small grey inset text bar next to each line, rather than retaining one longer grey inset text bar that spans the different lines of content).

Reply:
Like you want it to be a blank line? 
To visually separate them, you just need to have the `>` at the start of the line without any text on it (if that makes sense). 

```
> Paragraph 1
> 
> Paragraph 2
```

Reply:
Yep, exactly - thanks!